plugins/server/com.ucasoft/simple-memory-cache/2.0/documentation.md

```diff
@@ -0,0 +1,34 @@
+# Ktor Simple Memory Cache
```

Comment:
Could you drop the top header here to make it consistent with other plugin docs?

Reply:
Done.